src/test/common/fdc3.app-channels.ts

```diff
@@ -54,8 +54,8 @@ export function createAppChannelTests(cc: ChannelControl<any,any>, documentation
         const testChannel = await cc.createTestChannel()
         const resolveExecutionCompleteListener = cc.initCompleteListener(acTestId4)
         let receivedContext = false;
-        cc.setupAndValidateListener1(testChannel, "fdc3.instrument", errorMessage, () => { receivedContext = true })
-        cc.openChannelApp(acTestId4, null, APP_CHANNEL_AND_BROADCAST_TWICE)
+        await cc.setupAndValidateListener1(testChannel, "fdc3.instrument", errorMessage, () => { receivedContext = true })
```

Comment:
This util function is not adding a typed listener, which is the point of the test.